test/equalsStrict.js

```diff
@@ -4,24 +4,24 @@ const chai = require('chai');
 const testCases = require('../util/output/equalsTestCases.json');
 chai.should();
 
-describe('Equals test', function () {
+describe('Equals strict test', function () {
 
     it('should calculate equals correctly', function () {
```

Comment:
can you rename remaining "equals" texts to "equalsStrict" in this file? 


1. in it()
2. assertion message strings